kubernetes/kserve/kf_request_json/v2/bert/README.md

```diff
@@ -0,0 +1,115 @@
+# TorchServe example with Huggingface bert model
```

Comment:
@shrinath-suresh Let's add the links in the main README, the current links are broken and we need to clarify that here we have utilities to make the byte and tensor inputs.

Reply:
@shrinath-suresh Also, it would be great to clarify that for V1 there is image2byte utility. How V1 would work for BERT, I think it would be good to move bert.json, mnist.json and text_classifer.json to V1 and have a short README there as well.

Reply:
Added a step `Step - 5 : Generating input files` in the main readme with links for input file generation using utilities.

Reply:
Moved the json files to v1 folder and added a short readme - https://github.com/shrinath-suresh/serve/tree/ksserve-v2-kf-request-json/kubernetes/kserve/kf_request_json/v1